app.py

```diff
@@ -1,4 +1,4 @@
-# !pip install 'git+https://github.com/Lightning-AI/stablediffusion.git@lit'
+# !pip install 'git+https://github.com/Lightning-AI/stablediffusion.git@make_deepspeed_opt_in'
```

Comment:
To be reverted once https://github.com/Lightning-AI/stablediffusion/pull/6 is merged

Reply:
Removed.